hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/clean/CleaningTriggerStrategy.java

```diff
@@ -18,7 +18,14 @@
 
 package org.apache.hudi.table.action.clean;
 
+import org.apache.hudi.common.config.EnumDefault;
+import org.apache.hudi.common.config.EnumDescription;
+import org.apache.hudi.common.config.EnumFieldDescription;
+
+@EnumDescription("Controls when cleaning is scheduled")
 public enum CleaningTriggerStrategy {
```

Comment:
Only 1 option, but seems like we could add others pretty easily in the future. We could hide the config for now?

Reply:
Let's keep it.  It is meant for extensibility.